libs/wingsdk/src/std/resource.ts

```diff
@@ -174,6 +191,19 @@ export abstract class Resource extends Construct implements IResource {
     );
   }
 
+  /**
+   * "Lifts" a value into an inflight context. If the value is a resource (i.e. has a `_toInflight`
+   * method), this method will be called and the result will be returned. Otherwise, the value is
+   * returned as-is.
+   *
+   * @param value The value to lift.
+   * @returns a string representation of the value in an inflight context.
+   * @internal
+   */
+  protected static _lift(value: any): string {
```

Comment:
Maybe instead just make `liftObject()` available as a static method somewhere under `std` (`std.Inflight.liftObject()`).

Doesn't seem like there's too much value in exposing it from every resource in wing